1-2/소스.c

```diff
@@ -0,0 +1,8 @@
+﻿#include <stdio.h>
+
+int main(void)
+{
+	printf("%d에서 %d을 뺸 값은 %d입니다.",15 , 37, -22);
```

Comment:
```suggestion
	printf("%d에서 %d을 뺸 값은 %d입니다." ,15 , 37, -22);
```

위와 같이 수정되어야 할 것 같아요.